lib/vmdb/loggers.rb

```diff
@@ -33,6 +33,7 @@ def self.apply_config(config)
       apply_config_value(config, $datawarehouse_log,  :level_datawarehouse)
       apply_config_value(config, $fog_log,            :level_fog)
       apply_config_value(config, $gce_log,            :level_gce)
+      apply_config_value(config, $ibmcloud_log,       :level_ibmcloud)
```

Comment:
Can you name this `$ibm_cloud_log`?  That matches the provider name (`ManageIQ::Providers::IbmCloud`) and the logger class (`IbmCloudLogger`)